pkgs/development/tools/pnpm/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, fetchurl
+, nodejs
+, stdenvNoCC
+, testers
+, withNode ? true
```

Comment:
What's the reason for enabling/disabling this?

Reply:
It was enabled in the Yarn top-level package so I thought I should do the same for pnpm.